src/externalized/json/Json.h

```diff
@@ -0,0 +1,99 @@
+#pragma once
+
+#include "RustInterface.h"
+
+#include <string_theory/string>
+#include <vector>
+
+class JsonObject;
+
+class JsonValue {
+	public:
+		// Takes ownership of RJsonValue
+		JsonValue(RJsonValue* value) {
```

Comment:
You should not just write a comment about it but express it in code: `JsonValue(std::unique_ptr<RJsonvalue> && value)`.